src/sentry/integrations/message_builder.py

```diff
@@ -20,17 +20,17 @@
 
 
 def format_actor_options(
-    actors: Iterable[Team | RpcUser], use_block_kit: bool = False
+    actors: Sequence[Team | RpcUser], is_slack: bool = False
```

Comment:
this particular line -- the `Iterable` should have stayed -- probably a merge conflict sorry about that!

Reply:
oopsie, thanks for letting me know